### DISCUSSION TITLE
Installation Failed : Add-AppxPackage Deployment failed with HRESULT: 0x80Ø73CFD, A Prerequisite for an install could not be satisfied.

### DISCUSSION THREAD

**Discussion by NaiveInvestigator** [LSPosed/MagiskOnWSALocal#376] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/72654596/216332056-9147835d-56e8-48b1-9959-596e5b040afd.png)

I wasn't sure if this was a bug or not so that is why I've posted it here instead of the issues tab. This error occurs after I run the `Install.ps1` file. I have searched around for hours but sadly couldn't find anything that could fix this.

Windows Details:
Edition	Windows 11 Pro
Version	22H2
Installed on	‎27/‎01/‎2023
OS build	22623.1245
Experience	Windows Feature Experience Pack 1000.22638.1000.0

WSL Details:
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.1 LTS
Release:        22.04
Codename:       jammy

**Comment by Howard20181** [comment, marked as answer]
Copy to Windows path.

**Reply by NaiveInvestigator** [reply]
> Copy to Windows path.

What should to add to the windows path? I'm confused sorry.

**Reply by Howard20181** [reply]
> > Copy to Windows path.
> 
> What should to add to the windows path? I'm confused sorry.

Move the output file to Windows

**Reply by NaiveInvestigator** [reply]
> > > Copy to Windows path.
> > 
> > 
> > What should to add to the windows path? I'm confused sorry.
> 
> Move the output file to Windows

OMGGG TYSM for the help!!! Moving the `WSA_2211.40000.11.0_x64_Release-Nightly-with-magisk-25.2(25200)-stable-MindTheGapps-13.0-RemovedAmazon` folder from `output` to the `Windows` folder in my C drive worked!

On a side note too, rn i opened the Magisk app and tried to install magisk again there lol. It spouted some errors, would it like give any issues in the future?
